recipes/protobuf/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "3.23.1":
```

Comment:
This is not the correct version for C++.  Protobuf 23.x is 4.23.x in C++.  This is a major release, with many breaking changes.

https://protobuf.dev/news/2022-08-03/#cpp-changes

> we are planning a major version bump for C++. We plan to make some changes to the assets that we release starting with our 22.x release line.

Reply:
@coryan 
Thanks a lot!
I fixed versions in recipe.